git/showref.go

```diff
@@ -54,6 +54,18 @@ func (r Ref) TabString() string {
 	return fmt.Sprintf("%v\t%v", r.Value, r.Name)
 }
 
+// Gets the "ref name" of the ref suitable for showing in places
+//  like git log and git rev-list.
+func (r Ref) RefName() string {
```

Comment:
I would expect "RefName" method to return  `Refname(r.Name)`

Then you can add a ShortName` (or whatever git documentation refers to the short name as..) method to Refname and then use that.

Reply:
The git documentation calls these short form names ref names. See documentation of %D here: https://git-scm.com/docs/git-rev-list